### ISSUE TITLE
[Broken Website] www.google.com/search (Dynamic & Static Filter blocks page elements from showing)

### ISSUE THREAD

**Issue by SojournerOne** [darkreader/darkreader#6198]
# Broken Website
## Website
https://www.google.com/search?q=images&tbm=isch&ved=2ahUKEwjNlYLc087xAhX1AjQIHYT8CJcQ2-cCegQIABAA&oq=images&gs_lcp=CgNpbWcQAzIECAAQQzIFCAAQsQMyBQgAELEDMgUIABCxAzIICAAQsQMQgwEyAggAMgIIADICCAAyAggAMgIIADoHCAAQsQMQQ1D7ywJY9M8CYJXRAmgAcAB4AIABUYgBpgOSAQE2mAEAoAEBqgELZ3dzLXdpei1pbWfAAQE&sclient=img&ei=f2jkYI3aEvWF0PEPhPmjuAk&bih=885&biw=1920&client=firefox-b-d&sfr=vfe

## How to Reproduce
Go to www.google.com and perform a search. Make sure Dark Reader is set to Dynamic or Static Filter.

## Expected behavior
The elements of the page, including images and bounding boxes, should be visible.

## Actual behavior
Elements of the page are not visible.

## Screenshots
https://i.imgur.com/v9DFkMu.png

## System Information
- OS: Windows 8.1 Home Edition (64-bit)
- Browser: Firefox 89.0.2 (64-bit)
- Dark Reader Version: Dark Reader 4.9.33

## Additional context
Switching to Filter or Filter+ makes the pages visible.

**Comment by Gusted** [comment]
Hi @SojournerOne!

That seems to be an issue with the fallback style, looking at the screenshot.

Does this issue persist after a simple close/open of the browser?
Also does it still happen if you disable/enable dark reader on google?

Regards,
Gusted

**Comment by SojournerOne** [comment]
I came back after I discovered a mysterious extension lurking in my Firefix add-ons, one that i did not install. I removed it without double-checking it's home page so I can't be sure which one it was, but removing it did resolve the issue. Dark Reader was not the culprit after all.

I recently reinstalled my OS, and Firefox was recently reinstalled as well. It seems this interloper managed to somehow insinuate itself with the fresh install.
